x/auth/client/rest/rest_test.go

```diff
@@ -497,83 +436,6 @@ func (s *IntegrationTestSuite) testQueryIBCTx(txRes sdk.TxResponse, cmd *cobra.C
 	s.Require().Contains(errResp.Error, errMsg)
 }
 
-// TestLegacyRestErrMessages creates two IBC txs, one that fails, one that
```

Comment:
@amaurym how should I deal with these tests? Should I open an issue? Comment them out? Replace the IBC parts somehow?

Reply:
afaik it's only one test. If you're okay to move this one test to the IBC repo, that'd be ideal. If not, then just create an issue and we'll figure out later.

Reply:
I think there are a couple tests, but I'll look into moving them to the IBC repo and report back

Reply:
I managed to move these over to ibc-go but it involved some code duplication since we need to route the suite setup to the ibc-go simapp. 

I guess we cannot bring these tests back to the SDK until the SDK supports a Msg type that doesn't support amino and one that doesn't support REST?